homeassistant/components/rabbitair/entity.py

```diff
@@ -0,0 +1,62 @@
+"""A base class for Rabbit Air entities."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from rabbitair import Model
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import RabbitAirDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+MODELS = {
+    Model.A3: "A3",
+    Model.BioGS: "BioGS 2.0",
+    Model.MinusA2: "MinusA2",
+    None: None,
+}
+
+
+class RabbitAirBaseEntity(CoordinatorEntity[RabbitAirDataUpdateCoordinator]):
+    """Base class for Rabbit Air entity."""
+
+    def __init__(
+        self,
+        coordinator: RabbitAirDataUpdateCoordinator,
+        entry: ConfigEntry,
+    ) -> None:
+        """Initialize the entity."""
+        super().__init__(coordinator)
+        self._entry = entry
```

Comment:
The current config entry is already available in the coordinator: `self.coordinator.config_entry`, we don't need to store it here.

Reply:
Deleted that line. Looks like it was some code that was accidentally left behind and not even used.